UI/obs-app.cpp

```diff
@@ -1249,7 +1249,7 @@ bool OBSApp::SetTheme(std::string name, std::string path)
 bool OBSApp::InitTheme()
 {
 	defaultPalette = palette();
-	setStyle(new OBSIgnoreWheelProxyStyle());
+	setStyle(new OBSProxyStyle());
```

Comment:
This reverts an intentional design decision to stop users from accidentally scrolling combo boxes when trying to scroll the page they're in.

Reply:
It doesn't revert it (I was the one who wrote the code to stop users from wheel scrolling so I made sure that that doesn't happen :P).
I just renamed the ProxyStyle class since it's no longer exclusive to ignoring wheel scroll (so the old name would have been misleading), probably should have mentioned that in the PR though.